wcivf/apps/elections/tests/test_models.py

```diff
@@ -158,6 +158,30 @@ def test_friendly_name_mayoral(self, post_election):
             == "Greater London Authority mayoral election"
         )
 
+    @pytest.mark.django_db
+    def test_past_registration_deadline(self, post_election):
+        # election = ElectionWithPostFactory()
+        # election.past()
+        post = PostFactory(territory="ENG")
+        # territory = post.territory
+        oldest = PostElectionFactory(
+            ballot_paper_id="parl.cities-of-london-and-westminster.2019-05-06",
+            post=post,
+            election=ElectionFactoryLazySlug(
+                election_date="2019-5-6", current=False, election_type="parl"
+            ),
+        )
+        future = PostElectionFactory(
+            ballot_paper_id="parl.cities-of-london-and-westminster.2021-05-06",
+            post=post,
+            election=ElectionFactoryLazySlug(
+                election_date="2021-5-6", current=True, election_type="parl"
+            ),
+        )
+
+        assert oldest.past_registration_deadline is True
+        assert future.past_registration_deadline is False
```

Comment:
When we get past the registration deadline for this election (Is it 19th April?) this test will fail. We can get around this by using Freezegun (docs for reference https://pypi.org/project/pytest-freezegun/) which we already have installed and are using in other tests, to fix the date at the time the test runs to before the deadline.

Reply:
Example of other tests where we do this https://github.com/DemocracyClub/WhoCanIVoteFor/blob/64347d36f05df59fd9808eeb885932b4994a1912/wcivf/apps/parishes/tests/test_models.py#L10